secretsmanager/src/base64.cpp

```diff
@@ -36,7 +36,7 @@ string base64Enc(const vector<uint8_t>& msgBytes) {
 
```

Comment:
Wrap every variable which holds a resource with a smart pointer.
This is an example implementation:
https://stackoverflow.com/a/63711162